### ISSUE TITLE
It's weird that I ended up with a bunch of mosaics

### ISSUE THREAD

**Issue by guanzhenghua** [ubc-tea/SADM-Longitudinal-Medical-Image-Generation#1]
I use the following data set in the code, first run ACDC_loader, then ACDC_prapare, and finally run SADM, leaving the default parameters, and finally loss is 0.1691, visualized as a bunch of mosaics.
Thank you very much for the answer. This is very important to me.
You can also contact me by email. My email address is guanzhenghua2022@email.szu.edu.cn

**Comment by wltjr1007** [comment]
Current commit contains the minimal code for training and validation (i.e., hyperparameters are not the same as experiments in the paper). The full code will be ready before the camera-ready version of the paper is published (or hopefully sooner). Please stay tuned.

**Comment by guanzhenghua** [comment]
Thank you very much for your answer. 
Because of current commit only contains the minimal code for training and validation. I am worried about whether the results similar to those in the paper can be obtained just by adjusting hyperparameters. About using the dataset in the code. Could you give me some hints about hyperparameters. My gpu is Nvidia A6000 48GB. Thanks a lot.

**Comment by Barrett-python** [comment]
I have the same problem when I run the code.
